x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/job_type.tsx

```diff
@@ -17,64 +17,75 @@ interface Props {
   setFormState: React.Dispatch<React.SetStateAction<any>>;
 }
 
-export const JobType: FC<Props> = ({ type, setFormState }) => {
-  const outlierHelpText = i18n.translate(
-    'xpack.ml.dataframe.analytics.create.outlierDetectionHelpText',
-    {
-      defaultMessage: 'Outlier detection identifies unusual data points in the data set.',
-    }
-  );
+interface JobDetails {
+  [key: string]: {
+    helpText: string;
+    icon: string;
+    title: string;
+  };
+}
 
-  const regressionHelpText = i18n.translate(
-    'xpack.ml.dataframe.analytics.create.outlierRegressionHelpText',
-    {
+const jobDetails: JobDetails = {
+  [ANALYSIS_CONFIG_TYPE.OUTLIER_DETECTION]: {
+    helpText: i18n.translate('xpack.ml.dataframe.analytics.create.outlierDetectionHelpText', {
+      defaultMessage: 'Outlier detection identifies unusual data points in the data set.',
+    }),
+    icon: 'outlierDetectionJob',
+    title: 'Outlier detection',
```

Comment:
These card titles should be internationalized (looks like they aren't used in the `onClick` handling).

Reply:
Updated in 3c51fbaa5146278c93c81afbd2ca5f007cabb449